admin/tests/integration/components/organizations/all-tags_test.js

```diff
@@ -36,40 +36,11 @@ module('Integration | Component | organizations/all-tags', function (hooks) {
     this.set('model', { organization, allTags: [tag1, tag2, tag3, tag4] });
 
     // when
-    await render(hbs`<Organizations::AllTags @model={{this.model}} />`);
-
-    // then
-    assert.dom('.organization-all-tags-list__tag:first-child .pix-tag').containsText('AEFE');
-    assert.dom('.organization-all-tags-list__tag:last-child .pix-tag').containsText('POLE EMPLOI');
-  });
-
-  test('it should display an active tag when it is associate to an organization', async function (assert) {
-    // given
-    const tag1 = EmberObject.create({ id: 1, name: 'MEDNUM' });
-    const organizationTag1 = EmberObject.create({ name: 'MEDNUM' });
-    const organization = EmberObject.create({ tags: [organizationTag1] });
-
-    this.set('model', { organization, allTags: [tag1] });
-
-    // when
-    await render(hbs`<Organizations::AllTags @model={{this.model}} />`);
-
-    // then
-    assert.dom('.pix-tag--purple-light').exists();
-  });
-
-  test('it should display an inactive tag when it is not associate to an organization', async function (assert) {
-    // given
-    const tag1 = EmberObject.create({ id: 1, name: 'MEDNUM' });
-    const organization = EmberObject.create({ tags: [] });
-
-    this.set('model', { organization, allTags: [tag1] });
-
-    // when
-    await render(hbs`<Organizations::AllTags @model={{this.model}} />`);
+    const screen = await render(hbs`<Organizations::AllTags @model={{this.model}} />`);
 
     // then
-    assert.dom('.pix-tag--grey-light').exists();
+    assert.dom(screen.getAllByRole('button')[0]).hasText('AEFE');
+    assert.dom(screen.getAllByRole('button')[3]).hasText('POLE EMPLOI');
```

Comment:
On pourrait juste vérifier ici l'existence des boutons ?

```suggestion
    assert.dom(screen.getByRole('button', { name: 'AEFE' })).exists();
    assert.dom(screen.getByRole('button', { name: 'POLE EMPLOI' })).exists();
```

Reply:
On veut ici s'assurer de l'ordre alphabétique, si on s'assure qu'ils existent, on ne sait pas dans quel ordre ils s'affichent